index.bs

```diff
@@ -754,22 +768,28 @@ user consent to a specific transaction. The structure of these signatures is def
         unsigned long                        timeout;
         USVString                            rpId;
         sequence<ScopedCredentialDescriptor> allowList = [];
+        boolean                              verificationGesture;
```

Comment:
@balfanz: 
> Why do we need this in the getA call? It's just needed in makeC, right?

Correct, I think, in the sense of having the user's credential created using an authenticator with the RP's desired properties, at makeCredential time, and then the RP can stipulate that authenticator's usage at getAssertion time by putting it first in the allowList sequence.  Yes?